packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -637,12 +637,11 @@ const ArrayMixin = Mixin.create(Enumerable, {
   mapBy,
 
   /**
-    Returns an array with all of the items in the enumeration that the passed
-    function returns true for. This method corresponds to `filter()` defined in
-    JavaScript 1.6.
+    Returns a new array with all of the items in the enumeration that the provided
```

Comment:
I think instead of specifiying the version of JavaScript it became available in, we link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter instead?